languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -1661,11 +1661,14 @@ inherit .containing_class_value
 
 
 
-(field_definition
-  property:(property_identifier)@property)@field_def {
-
+(field_definition)@field_def {
   node @field_def.after_scope
   node @field_def.before_scope
+}
+
+(field_definition
+  property:(_)@property)@field_def {
```

Comment:
Second (gah, this diff is unsemantic), we generalize the rule to work for any property, regardless of node type, to accommodate private properties and whatever else JS and the grammar might throw at us in the future.